src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -320,7 +321,7 @@ def test_acckernelsdirective_gencode(default_present):
         "      !$acc end kernels\n" in code)
 
 # (1/1) Method gen_code
-@pytest.mark.parametrize("async_queue", [False, 1, Signature('stream1')])
+@pytest.mark.parametrize("async_queue", [False, 1, 0, Reference(Symbol('stream1'))])
```

Comment:
It's mainly to exercise the problem that I identified earlier. If (for some reason) a user decides to store the stream/queue number in an array element then it would be specified as e.g. `stream2(1)`

Reply:
Ah, ok, I see what you mean, I miss-understood the line. I will check.